NetKAN/Mk2Expansion.netkan

```diff
@@ -7,14 +7,14 @@
         { "name": "KlockheedMartian-Gimbal" },
         { "name": "ModuleManager", "min_version": "2.5.3" },
         { "name": "CommunityResourcePack" },
-        { "name": "InterstellarFuelSwitch" }
+        { "name": "InterstellarFuelSwitch-Core" }
     ],
     "suggests": [
 	{ "name": "FerramAerospaceResearch" }
     ],
     "install" : [
         {
-            "find"       : "Mk2Expansion",
+            "find"       : "GameData/Mk2Expansion",
```

Comment:
But not Mk2_Stock-a-like_Expansion-1.6.05.zip\**Mk2Expansion**\GameData\Mk2Expansion
(1.6.05 was taken down not to break too many installs, but it might reoccur in the future)